pyccel/stdlib/cwrapper/meson.build

```diff
@@ -0,0 +1,10 @@
+
+
+py_dep = py.dependency()
```

Comment:
I assume that `py` is defined elsewhere? The `meson-python` [tutorial](https://mesonbuild.com/meson-python/tutorials/introduction.html#the-meson-build-description) suggests that this line should exist somewhere:
```python
py = import('python').find_installation(pure: false)
```

Reply:
In particular the command that is printed is:
`py = import('python').find_installation('/usr/bin/python3.13', modules: ['numpy'])`
This ensures that the correct version of Python is used (e.g. on my laptop I have python 3.9-3.12 installed so I need to make sure I generate the extensions for the right version)